keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/verification/TestContainerVerificationEvaluator.kt

```diff
@@ -66,17 +66,16 @@ class TestContainerVerificationEvaluator(
         taskLauncher.submitJob(
           type = VERIFICATION,
           subject = "container integration test for ${context.deliveryConfig.application}.${context.environmentName}",
-          description = "Verifying ${context.version} in environment ${context.environmentName} with test container ${verification.repository}/${verification.tag}",
+          description = "Verifying ${context.version} in environment ${context.environmentName} with test container ${verification.repository}:${verification.tag}",
```

Comment:
Nit: note that the `TestContainerVerification.id` field uses a slash to delimit repository and tag.

```
  override val id = "$repository/$tag"
```

Would be good to be consistent.

Reply:
Good spot. I changed it because of the way Deck renders it in the run job stage configuration view. I'll change it there as well so it's consistent in all 3 places.